torch/csrc/distributed/c10d/reducer.cpp

```diff
@@ -155,15 +153,15 @@ Reducer::Reducer(
   // used in an autograd pass. If they are not, we know their grad tensors
   // can be marked as ready for reduction.
   {
-    const auto replica_count = replicas_.size();
+    const auto replica_count = 1;
     grad_accumulators_.resize(replica_count);
```

Comment:
can be done in followup PR: can `grad_accumulators_` also be simplified to `std::vector<std::shared_ptr<torch::autograd::Node>>` and get rid of `replica_count `?

Reply:
Done